dev.zed.Zed.metainfo.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>dev.zed.Zed</id>
+  <name>Zed</name>
+  <summary>High-performance, multiplayer code editor from the creators of Atom and Tree-sitter.</summary>
+  <description>
+    <p>
+      Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. It combines the power of an IDE with the responsiveness of a lightweight editor for productivity you can feel under your fingertips.
+    </p>
+    <p>
+      Zed leverages every CPU core and your GPU to start instantly, load files in a blink, and respond to your keystrokes on the next display refresh. It supports GitHub Copilot out of the box, and you can use GPT-4 to generate or refactor code with natural language prompts.
+    </p>
+    <p>
+      Zed maintains a full syntax tree for every buffer, enabling precise code highlighting, auto-indent, a searchable outline view, and structural selection. It also supports the Language Server Protocol for autocompletion, code navigation, diagnostics, and refactorings.
+    </p>
+    <p>
+      Zed integrates a full-featured terminal, Vim-style modal editing, and a variety of light and dark themes. Its collaborative features include shared workspaces, real-time navigation, and a virtual office for software teams with channels for discussion, planning, and project tracking.
+    </p>
+    <p>
+      Zed's performance is driven by the GPUI framework, multi-core support with Rust, and extensive fuzz testing for stability. Its CRDT-based buffers provide a foundation for multiple forms of collaboration.
+    </p>
+    <p>
+      NOTE: This wrapper is not yet verified by, affiliated with, or supported by the developers of Zed.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://zed.dev/_next/image?url=%2F_next%2Fstatic%2Fmedia%2Fhero-1.bb06921e.png&amp;w=2048&amp;q=75</image>
```

Comment:
These screenshots are from MacOs. https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines#take-screenshots-on-linux